templates/foreman_gutterball.yaml.erb

```diff
@@ -0,0 +1,2 @@
+<%# for foreman_gutterball plugin %>
```

Comment:
@ehelms is it ok for url to fall into something like :foreman_gutterball_common:
i think if the 2 settings use the same namespace, one will overwrite the other instead of merging them.